win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -731,29 +729,6 @@ SetDIBitsToDevice(
 
     cjBmpScanSize = DIB_BitmapMaxBitsSize((LPBITMAPINFO) lpbmi, ScanLines);
 
-    pvSafeBits = RtlAllocateHeap(GetProcessHeap(), 0, cjBmpScanSize);
-    if (pvSafeBits)
-    {
-        _SEH2_TRY
-        {
-            RtlCopyMemory(pvSafeBits, Bits, cjBmpScanSize);
-        }
-        _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
-        {
-            Hit = TRUE;
-        }
-        _SEH2_END
-
-        if (Hit)
-        {
-            // We don't die, we continue on with a allocated safe pointer to kernel
-            // space.....
-            DPRINT1("SetDIBitsToDevice fail to read BitMapInfo: %p or Bits: %p & Size: %u\n",
-                pConvertedInfo, Bits, cjBmpScanSize);
-        }
-        DPRINT("SetDIBitsToDevice Allocate Bits %u!!!\n", cjBmpScanSize);
-    }
-
     if (!GdiGetHandleUserData(hdc, GDI_OBJECT_TYPE_DC, (PVOID) & pDc_Attr))
     {
         SetLastError(ERROR_INVALID_PARAMETER);
```

Comment:
This failure code path is leaking the allocated `pConvertedInfo` (and was leaking previously the `pvSafeBits` as well).
Other code above did, instead of the `return 0;` the following:
```c
    LinesCopied = 0;
    goto Exit;
```